src/plugins/home/public/services/section_type/section_type.ts

```
@@ -127,10 +127,7 @@ export class SectionTypeService {
             }
 
             if (heroes || sections) {
-              homepage
-                .save({})
-                .then(() => error$.next(undefined))
-                .catch((e) => error$.next(e));
+              homepage.save({});
```

is there anything that tracks fixing this permanently?

---

I could open an issue. Currently we are using this tmp one for unblocking purpose.

---

actually we could keep the issue open and revisit to see if there is better solution

---

Hi Anan. Will this fix have some potential issue as it swallowed the error. Will it show information that not suppose to show to user?

---

Maybe we could leave a comment here to describe why it's crashed(I'm curious ;)

---

+1